scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1694,6 +1653,9 @@ function initialize_pxb_commands()
         disver="--no-version-check"
     fi
 
+    # KH: TODO: Just for tests, as we don't have PXB 8.4 yet. Remove it
+    disver+=" --no-server-version-check"
```

Comment:
PXB-8.4 is already released. Does still comment still hold good?

Reply:
done. Leftover from the time when pxb 8.4 was not ready yet